### ISSUE TITLE
Finetuning the old model ??

### ISSUE THREAD

**Issue by msiraj83** [jiny2001/dcscn-super-resolution#22]
is it is possible to finetued the old model  ? Or we have to put all the images for the old model along with our own images and train the model from scratch ? Can any body guide me in this regards ?

**Comment by jiny2001** [comment]
Hi, 
If you use --load_model_name, you can load already trained model for initial value. And then you can train with your own dataset with smaller learning rate? This would be an easy way.

If you want to train only specific layer's weights and biasses during your second training, you should modify the code for optimizer. I know how to do that, but is a little tricky. You should better rename the layers depend on which layers you want to fine tune.

**Comment by msiraj83** [comment]
Dear @jiny2001 

Thanks for your information. For example i want to finetuned Yan91 model on my dataset siraj83

`python train.py --dataset siraj83 --load_model_name yan91 --filters 128 --layers 10 --initial_lr 0.00001`

 Is i am right??

**Comment by jiny2001** [comment]
Hi,

Sorry, there was a bug in my script. Could you update to the latest one?

And then, after you trained like below, 
`python train.py --dataset yang91 --filters 128 --layers 10`

The  output model filename would be displayed on the console. In this case, model filename would be "dcscn_L10_F128to48_NIN_A64_PS_R1F32". 

```
DCSCN v2-------------------------------------
2018/4/28 11:5:24 [dcscn_L10_F128to48_NIN_A64_PS_R1F32]
```

You can load it as a initial value like below
`python train.py --dataset=your_dataset_file_dir --filters=128 --layers=10 --load_model_name=dcscn_L10_F128to48_NIN_A64_PS_R1F32`

**Comment by msiraj83** [comment]
Dear  @jiny2001 

Thanks. I want to fine-tuned the your train model and you suggest that use the low learning rate. What is the default learnig rate and how i speicfy the new learning rate.. I think it will be like this ??

`python train.py --dataset=your_dataset_file_dir --filters=128 --layers=10 --load_model_name=dcscn_L10_F128to48_NIN_A64_PS_R1F32 --initial_lr 0.0001`

Could you guide me in this regards

Thanks

**Comment by jiny2001** [comment]
Hi, the default starting learning rate (initial_lr) is 0.002.
Using smaller starting learning rate is just one idea. You also need to think about lr_decay, lr_decay_epoch, end_lr to control learning rate.

Also optimizer and number of training iterations (training_images) should be important.

Or you can freeze some layers' weights and biases. Since the best parameters are depend on how much/different data you use, I can't advise you which parameter to change.

**Comment by jiny2001** [comment]
Hi, I'm gonna close this issue since it's like a question for now. Feel free to ask/open again!
